lib/src/ToggleSelector/ToggleSelectorItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import classnames from "classnames";
+import { randomId } from "@sebgroup/frontend-tools";
+
+export type ToggleSelectorItemProps = Omit<JSX.IntrinsicElements["input"], "type" | "value"> & {
+    label: React.ReactNode;
+    icon?: React.ReactNode;
+    value: string;
+};
+
+export type ToggleSelectorItemComponent = React.FC<ToggleSelectorItemProps>;
+
+export const ToggleSelectorItem: ToggleSelectorItemComponent = ({ className, label, icon, ...props }: ToggleSelectorItemProps) => {
+    const [id, setId] = React.useState<string>(props.id);
+
+    const onKeyDown = React.useCallback((event: React.KeyboardEvent<HTMLDivElement>) => {
+        if (props.disabled) {
+            return;
+        }
+        switch (event.keyCode) {
+            case 32:
+            case 13:
+                (event.target as HTMLDivElement)?.previousElementSibling?.dispatchEvent(new MouseEvent("click", { bubbles: true }));
+                break;
+            default:
+                break;
+        }
+    }, []);
```

Comment:
I'm not sure why you need this, supposedly, native input element should behave correctly when keyboard interactions are introduced